Makefile

```
@@ -117,7 +117,7 @@ setup/moq:
 
 .PHONY: setup/service_account/oc_login
 setup/service_account/oc_login:
-	@oc login --token=$(shell sh -c "oc serviceaccounts get-token rhmi-operator -n ${NAMESPACE}") --server=$(shell sh -c "oc whoami --show-server") --kubeconfig=TMP_SA_KUBECONFIG --insecure-skip-tls-verify=true
+	@oc login --token=$(shell sh -c "oc create token rhmi-operator") --server=$(shell sh -c "oc whoami --show-server") --kubeconfig=TMP_SA_KUBECONFIG --insecure-skip-tls-verify=true
```

These new tokens have a fixed duration of 3600s by default, so the operator stops working after an hour - this could be a little inconvenient, perhaps setting 24h is preferable, or perhaps even longer? I'm unsure what / if there is a downside to a longer-lasting token.

I also I don't see any harm in specifying the namespace explicitly given the SA is in that namespace instead of assuming we're in it

```suggestion
	@oc login --token=$(shell sh -c "oc create token rhmi-operator -n ${NAMESPACE} --duration=24h") --server=$(shell sh -c "oc whoami --show-server") --kubeconfig=TMP_SA_KUBECONFIG --insecure-skip-tls-verify=true
```

---

Hmm, excellent points. 
There is no harm in restricting token to only operator namespace and we should, indeed, allow reconciler to run for more than an hour. Since this token will be only for a local running reconciler and each restart of the reconcile loop will regenerate it - 24 hours is more than enough. Can't imagine someone leaving it running over a night, but I'll drop the question in the general chat - maybe I'm missing something there.

---

> There is no harm in restricting token to only operator namespace

It's not actually restricting the token - the create token command is creating a token for the already existing service account named rhmi-operator so if you switch to a different project after running make cluster/prepare/local this command will fail until you switch back to the correct project.

Yep I think 24h should be fine. I've sometimes left it running overnight but not for any particular reason. The only thing that comes to mind is memory profiling but I'm unsure we do that and I guess that could be done via OLM deploy.